test/test_view.py

```diff
@@ -47,7 +50,7 @@ def test_populate_preview_image(self):
         with open(POPULATE_PREVIEW_IMAGE, 'rb') as f:
             response = f.read()
         with requests_mock.mock() as m:
-            m.get(self.baseurl + '/workbooks/3cc6cd06-89ce-4fdc-b935-5294135d6d42/'
+            m.get(self.siteurl + '/workbooks/3cc6cd06-89ce-4fdc-b935-5294135d6d42/'
```

Comment:
It was motivated by the taggedresource change.  For most of our other endpoints, the baseurl is "....site/siteid/resource", for example "...site/siteid/datasources".  Views didn't match this, and as best I could tell this was because of the off logic needed for the populate preview image call, which gets info from the workbooks endpoint instead of the views endpoint.  I thought it would be cleaner to have the Views endpoint behave like the others endpoints.